docs/cli-ref.rst

```diff
@@ -35,3 +35,48 @@ development and testing.
    :module: airflow.cli.cli_parser
    :func: get_parser
    :prog: airflow
+
+Environment Variables
```

Comment:
Curious why this is placed in cli-ref.rst -- any particular reason ?

Reply:
It is common practice to include a list of the environment variables supported next to a command list. It makes sense from the user's perspective. If the user executes commands, they also has shell access to set environment variables.
Examplle: 
https://www.sphinx-doc.org/en/master/man/sphinx-build.html#environment-variables
https://docs.python.org/3/using/cmdline.html

Reply:
We should change the title from Cli Ref to Cli Ref and Environment Variables like that Python docs link